query-engine/core/src/metrics/mod.rs

```diff
@@ -36,21 +36,27 @@ pub use registry::MetricRegistry;
 use serde::Deserialize;
 use std::sync::Once;
 
+pub(crate) const PRISMA_CLIENT_QUERIES_TOTAL: &str = "prisma_client_queries_total";
+pub(crate) const PRISMA_CLIENT_QUERIES_HISTOGRAM_MS: &str = "prisma_client_queries_duration_histogram_ms";
+
 // At the moment the histogram is only used for timings. So the bounds are hard coded here
 // The buckets are for ms
 pub(crate) const HISTOGRAM_BOUNDS: [f64; 10] = [0.0, 1.0, 5.0, 10.0, 50.0, 100.0, 500.0, 1000.0, 5000.0, 50000.0];
 // We need a list of acceptable metrics we want to expose, we don't want to accidentally expose metrics
 // that a different library have or unintended information
 const ACCEPT_LIST: &[&str] = &[
-    "pool_active_connections",
-    "pool_idle_connections",
-    "pool_wait_count",
-    "pool_wait_duration_ms",
-    "query_total_elapsed_time_ms",
-    "query_total_queries",
-    "query_active_transactions",
-    "query_total_operations",
-    "query_operation_total_elapsed_time_ms",
+    "prisma_pool_connections_opened_total",
+    "prisma_pool_connections_closed_total",
+    "prisma_pool_connections_open",
+    "prisma_pool_connections_busy",
+    "prisma_pool_connections_idle",
+    "prisma_client_queries_wait",
+    "prisma_client_queries_wait_histogram_ms",
+    "prisma_datasource_queries_duration_histogram_ms",
+    "prisma_datasource_queries_total",
+    "prisma_client_queries_active",
+    PRISMA_CLIENT_QUERIES_HISTOGRAM_MS,
+    PRISMA_CLIENT_QUERIES_TOTAL,
```

Comment:
Why are these two different?

Reply:
They are the only ones I could make a standard static variable. The rest are scattered across libraries or code and are difficult to make a consistent variable.

Reply:
Shouldn't we pull those in a separate crate if they're scattered across the codebase? That'd make it very easy to reuse everywhere?